dist/.gitattributes

```diff
@@ -1 +1,2 @@
 * text=auto
+.htaccess linguist-vendored
```

Comment:
@drKraken This should be in the main [`.gitattributes`](https://github.com/h5bp/html5-boilerplate/blob/master/.gitattributes.gitattributes) files as `dist/.htaccess linguist-vendored`.